pages/details/[id].js

```diff
@@ -24,6 +26,8 @@ export default function Details() {
   const destinationName = destination?.name || "Not found";
   const destinationQueryName = destinationName.replaceAll(" ", "-");
 
+  const calculateDuration = () => (destination?.endDate - destination?.startDate) / 86400;
```

Comment:
Just asking because I really don't know what is best here:
This is another method for calculating the duration. Does the user expect this to be the number of nights? If they travel less than 24 hours overnight, they would see 0 nights, correct?

Reply:
Since the user can only enter start date and end date, theres no hourly counting. If the user returns on the same day, there will be a duration of 0 nights.